src/controllers/simplified-account/settings/webhooks/detail/detail.controller.js

```diff
@@ -5,50 +5,46 @@ const formatSimplifiedAccountPathsFor = require('@utils/simplified-account/forma
 const { constants } = require('@govuk-pay/pay-js-commons')
 
 async function get (req, res) {
-  const status = (req.query.deliveryStatus === 'successful' || req.query.deliveryStatus === 'failed')
-    ? req.query.deliveryStatus
-    : undefined
-  const page = parseInt(req.query.page || 1)
   const baseUrl = formatSimplifiedAccountPathsFor(paths.simplifiedAccount.settings.webhooks.detail, req.service.externalId, req.account.type, req.params.webhookId)
+  const page = parseInt(req.query.page || 1)
   if (Number.isNaN(page) || page < 1) {
     res.redirect(baseUrl)
   }
 
+  const deliveryStatus = (req.query.deliveryStatus === 'successful' || req.query.deliveryStatus === 'failed') ? req.query.deliveryStatus : 'all'
   const webhook = await webhooksService.getWebhook(req.params.webhookId, req.service.externalId, req.account.id)
-  const messages = await webhooksService.getWebhookMessages(req.params.webhookId, { page, ...status && { status } })
-  const totalPages = Math.ceil(messages.total / 10)
+  const signingSecret = await webhooksService.getSigningSecret(req.params.webhookId, req.service.externalId, req.account.id)
+  const webhookMessages = await webhooksService.getWebhookMessages(req.params.webhookId, { page, ...deliveryStatus && { status: deliveryStatus } })
```

Comment:
What would make these even more readable is to make `webhooksService.<method>` return a defined class, and add JSDoc to it. See [this](https://github.com/alphagov/pay-selfservice/pull/4482/files#diff-5782efcd49320ce376425a52dddaed3fcaf639432b41d33e40b384d992002818R16-R30) an example. However you're not doing anything worse at the moment so it doesn't block approval of this PR, but it's something you may want to consider doing.